go.mod

```diff
@@ -95,6 +95,8 @@ require (
 	upper.io/db.v3 v3.7.1+incompatible
 )
 
+require github.com/pkg/errors v0.9.1 // indirect
```

Comment:
Why we need change this?

Reply:
I changed testify to 1.7.1 and it appears automaticllay.